app/views/home/maintainers.md

```diff
@@ -0,0 +1,45 @@
+> Audience: "developers", meaning anyone who has skills that are relevant to software development, that identify as a "maintainer" regularly contribute to an open source project.
+
+> What are you asking me to do?
+
+Take one day this week to make your project easier to use and more welcoming to new contributors.
+
+> Why should I care?
+
+You could use help
+
+> What are the reasons I should care about getting more contributors for my open source project?
+
+- **Reduce your workload** - Regular contributors help
```

Comment:
Agreed. Also: automating manual workflows, documenting and reducing scope and optimising for the 99% not the 100%.